src/autotype/xcb/AutoTypeXCB.cpp

```diff
@@ -404,10 +414,27 @@ bool AutoTypePlatformX11::GetKeycode(KeySym keysym, int* keycode, int* group, un
         }
     }
 
+    // try to find the best dead key mapping if we're unlucky to have one
+    if (!desc) {
+        for (const auto& map : deadMap) {
+            if (map.first == keysym) {
+                for (const auto& key : m_keymap) {
+                    if (key.sym == map.second) {
+                        if (desc == nullptr || key.group == *group) {
+                            desc = &key;
+                            isDead = true;
```

Comment:
Should break from the loops here

Reply:
Can't do that, sir. The loops are searching for the best match, not the only one like the loops above.

Reply:
Add a comment to that effect then "Do not break here since there may be a better match"